content/downloads.html

```diff
@@ -109,10 +109,10 @@ <h5 class='toc'>Current release</h5>
 	
 	<h5 class='toc'>Verifying file integrity</h5>
 	<p>
-		Instructions on verifying the integrity of the downloaded files can be found <a class='doclink' href='https://www.apache.org/info/verification.html'>here</a>.
+		How to <a class='doclink' href='https://www.apache.org/info/verification.html'>verify downloaded files</a>.
 	</p>
 	<p>
-		The KEYS file can be found <a class='doclink' href='https://www.apache.org/dist/juneau/KEYS'>here</a>.
+		<a class='doclink' href='https://www.apache.org/dist/juneau/KEYS'>Download KEYS </a>file.
```

Comment:
Download <a class='doclink' href='https://www.apache.org/dist/juneau/KEYS'>KEYS</a> file.

The link text needs to be 'KEYS' only please.

Reply:
I have made the change and created another PR since this PR was closed. 
https://github.com/apache/juneau-website/pull/8